features/admin_journey.feature

```diff
@@ -13,18 +13,18 @@ Scenario: As an admin user I wish be able to log in and to log out of Digital Ma
 @logout
 Scenario: As an admin user who has logged in to Digital Marketplace, I wish to search for a service
```

Comment:
If you add the tag then I'll fix the Jenkins job to skip those tests when running against production.